subsys/bluetooth/audio/Kconfig.has

```diff
@@ -55,6 +55,41 @@ config BT_HAS_HEARING_AID_TYPE
 	help
 	  The value shall be one of 3 defined by the HAS 1.0 specification table 3.2
 
+config BT_HAS_PRESET_COUNT
+	int "Preset record list size"
+	default 2
+	range 0 255
+	help
+	  This option sets the number of Hearing Access Service Presets
+	  that can be registered. Setting this value to 0 disables Hearing Aid Presets support.
```

Comment:
"Aid" -> #"Access" - better be consistent (and consistent with the spec).

Reply:
done